### ISSUE TITLE
Draft standardized worflow for manual projects in Smartling

### ISSUE THREAD

**Issue by gueroJeff** [mozilla-l10n/pm-projects#249]
Each content type has a spreadsheet file containing instructions for the project preparers and the l10n PMs. This content should be migrated to Mana (for project preparers) and the [l10n-drivers](https://github.com/mozilla-l10n/documentation/) documentation for the L10n PMs, in order to maintain documentation consistency.

- [x] Email
- [x] Snippets
- [x] Surveys
- [x] Legal markdown
- [x] Blog
- [x] SUMO articles
- [x] App store

Each content type should have its own page in mana and the l10n doc repo. These are some of the information that this needs to include. The audience is the l10n-driver PM team as well as anyone of our internal clients looking to understand how the process works. 

- Content type files templates
- Pre-processing needs
- Workflow within Smartling
- - Manual (intake -> execution -> delivery -> internal review -> finalization - review feedback to Smartling?) -
- Filtering and parsing (regex)
- Post-processing needs
- Review feedback to update TM
- Archival

**Comment by peiying2** [comment]
[Draft](https://docs.google.com/document/d/1PSCBH8Y-uQq5PRuvAmy2p1OalT-a6e6kUmLBhJA5wPg/edit) of the overall process. The non-standard internal review process and feedback loop need to be tested and confirmed.

**Comment by peiying2** [comment]
@gueroJeff please +r when you get chance. See if the flow is clearly described. The doc covers the area that the content-type process doesn't covered, such as filing the bug, Mozilla review process.

**Comment by peiying2** [comment]
Also, the Legal markdown is fully supported and it is a matter of setting up the connector with the legal repo. 

However, as I mentioned in the issue for Legal template, there are two types of legal requests, one is through the Legal repo, another is a Word file based one-off request. It usually is limited to a few locales for 1) internal consumption; 2). breach alert.

**Comment by gueroJeff** [comment]
Thanks Pei. I know you were waiting on a template from me last week. Sorry I wasn't able to provide one by Friday. This morning, I started writing up [a template that we can copy/paste into Mana](https://docs.google.com/document/d/1RDfv4yTt989lssrYhoM_LjwW8jkzXc8UG_niu-0Grkk/edit#)  for each content type page. I'll keep working on it tonight and tomorrow. What do you think?

This is a good start in terms of content. I think we need to make some changes though to reflect what we'd discussed about having one Mana page per content type linked from the Localization parent Mana page. We'll need to make sure it only contains information aimed at project preparers. 

On the other side, we'll need to adapt these steps for the l10n-driver documentation. Maybe what we can do is publish the requestor doc first and ask flod/delphine if they want their documentation structured in the same way (parent page -> children pages by content type). What do you think of that?

**Comment by peiying2** [comment]
Yes, agreed. I feel like we need to have basic feedback on structure/organization on the content, and then you and I can focus on the accuracy of the content, and fill in the info based on the structure.

**Comment by gueroJeff** [comment]
Awesome. Let's use my doc as the template for mana and your doc as the templates for l10n-drivers doc. Take a look at my [Mana page template](https://docs.google.com/document/d/1RDfv4yTt989lssrYhoM_LjwW8jkzXc8UG_niu-0Grkk/edit#) tomorrow and let me know what you think.

**Comment by gueroJeff** [comment]
[Here's an example of the Mana template applied in Mana.](https://mana.mozilla.org/wiki/display/FIREFOX/Request+a+blog+article+translation)

**Comment by gueroJeff** [comment]
@peiying2 I'm creating all of the Mana pages and migrating the content for each. I've been combining snippets + in-app messages into one category, and I see now that you have them split out. Looking at the templates, they're effectively the same structure and use the same directives. Is there any reason to not combine them into one content type and one set of instructions/template?

**Comment by peiying2** [comment]
> @peiying2 I'm creating all of the Mana pages and migrating the content for each. I've been combining snippets + in-app messages into one category, and I see now that you have them split out. Looking at the templates, they're effectively the same structure and use the same directives. Is there any reason to not combine them into one content type and one set of instructions/template?

They are not and should stay separate:
- Snippet, at project level (or job on Smartling) has one standard character limitation. 
- In-app messages have multiple character limits - 1) Subject and 2) short description (both counts one cell/string limit) and 3). long description (the total of multiple cells/strings count). This one is very important for the mobile message. Take a look of the template.

**Comment by gueroJeff** [comment]
> > @peiying2 I'm creating all of the Mana pages and migrating the content for each. I've been combining snippets + in-app messages into one category, and I see now that you have them split out. Looking at the templates, they're effectively the same structure and use the same directives. Is there any reason to not combine them into one content type and one set of instructions/template?
> 
> They are not and should stay separate:
> 
>     * Snippet, at project level (or job on Smartling) has one standard character limitation.
> 
That sounds like a difference for L10n PgMs though, not project requestors.
>     * In-app messages have multiple character limits - 1) Subject and 2) short description (both counts one cell/string limit) and 3). long description (the total of multiple cells/strings count). This one is very important for the mobile message. Take a look of the template.

I'm looking at both templates, and they're nearly the same (only observable difference is in cell D8 between the two templates). In the ["push notifications" template](https://docs.google.com/spreadsheets/d/1xQjzTZHdyfDbX2E8eBo289p-oY6xX_kAmBVQnmorto8/edit#gid=0), I'm not seeing the points you mention here that demonstrate the multiple character limits function unique to in-app messages. What should I be looking at instead?

**Comment by peiying2** [comment]
You can remove the Push notification. This is asked a lot, but we can use Snippet template for it.

Snippet should be labeled as Snippet because it can be used on mobile or desktop. 

For the in-app mobile prod description template: https://docs.google.com/spreadsheets/d/1H9BnzK7AfY0a3LFARM3Rede7_0RBrn4Brs3mOHuZNtU/edit#gid=1218117366 - check rows 12 and 13. Each cell contains multiple strings. A column counts the characters in Source (also to confirm it is within the character limit shown in B), B column shows the limit for the target language. 

Snippet/Push notification templates are much simpler. Usually a sentence or a short paragraph. Since I get requests for Push notification, we may want to include it in the title so requester knows where to look for the template.

**Comment by gueroJeff** [comment]
I think I see the confusion now. I’m talking about in-app messages, not App
Store descriptions. I have those noted as being distinct. Thanks for
clarifying that. It looks like we’re all good for this in Mana.
